elasticsearch-core/src/main/scala/com/sumologic/elasticsearch/restlastic/RestlasticSearchClient.scala

```diff
@@ -183,7 +190,7 @@ class RestlasticSearchClient(endpointProvider: EndpointProvider, signer: Option[
                       endpoint: String,
                       method: HttpMethod = POST,
                       query: UriQuery = UriQuery.Empty)
-                     (implicit ec: ExecutionContext): Future[RawJsonResponse] = {
```

Comment:
Oh good. lgtm

On Fri, Jun 24, 2016, 6:46 PM Kumar Avijit <notifications@github.com> wrote:

> In
> elasticsearch-core/src/main/scala/com/sumologic/elasticsearch/restlastic/RestlasticSearchClient.scala
> <https://github.com/SumoLogic/elasticsearch-client/pull/35#discussion_r68481885>
> :
>
> > @@ -183,7 +190,7 @@ class RestlasticSearchClient(endpointProvider: EndpointProvider, signer: Option[
> >                        endpoint: String,
> >                        method: HttpMethod = POST,
> >                        query: UriQuery = UriQuery.Empty)
> > -                     (implicit ec: ExecutionContext): Future[RawJsonResponse] = {
>
> So this works, e.g.:
>
> scala> class A {
>      |   def foo(x: Int)(implicit name: String = "foo"): Int = {
>      |     println (name)
>      |     x + 1
>      |   }
>      | }
> defined class A
>
> scala> new A().foo(0)
> foo
> res3: Int = 1
>
> scala> new A().foo(0)("bla")
> bla
> res4: Int = 1
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/SumoLogic/elasticsearch-client/pull/35/files/0b66aa5ee7d878d58da24e26b0d29aac46d48ffc#r68481885>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAeFZ7xYxp6Ho_M24gFOAglgmdC_uWk7ks5qPIh2gaJpZM4I-M_m>
> .
>